packages/@lwc/synthetic-shadow/src/faux-shadow/traverse.ts

```diff
@@ -122,6 +129,10 @@ export function getNodeOwner(node: Node): HTMLElement | null {
     return nodeOwner as HTMLElement;
 }
 
+export function isSyntheticSlotElement(node: Node): node is HTMLSlotElement {
+    return isSlotElement(node) && KEY__SHADOW_TOKEN_PRIVATE in node;
```

Comment:
`KEY__SHADOW_TOKEN_PRIVATE` just seemed like a convenient way to distinguish between slots created outside and inside LWC. Happy to choose another technique if needed.

Reply:
I think that it can be replaced by `isNodeShadowed`.

Reply:
Done